rcl/include/rcl/node.h

```diff
@@ -137,7 +137,7 @@ rcl_get_zero_initialized_node(void);
  * \return #RCL_RET_OK if the node was initialized successfully, or
  * \return #RCL_RET_ALREADY_INIT if the node has already be initialized, or
  * \return #RCL_RET_NOT_INIT if the given context is invalid, or
```

Comment:
if i am not mistaken, you mean that this line is the confusion, right? context is one of the arguments, so that it should return `RCL_RET_INVALID_ARGUMENT`?

if that is the case, how about trying to address with following doc?

```suggestion
 * \return #RCL_RET_NOT_INIT if the given context is not initialized, or
```

could it be less confusion?